src/test/java/com/pharmacy/Pharmacy/TestData.java

```diff
@@ -0,0 +1,61 @@
+package com.pharmacy.Pharmacy;
+
+import com.pharmacy.Pharmacy.dto.PharmacyBranchDTO;
+import com.pharmacy.Pharmacy.model.Pharmacy;
+import com.pharmacy.Pharmacy.model.PharmacyBranch;
+import com.pharmacy.Pharmacy.model.PharmacyBranchAddress;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class TestData {
+
+    List<Pharmacy> pharmacies = createPharmacies();
```

Comment:
Klasa jest okej. Można to zrobić w ten sposób. Dobrze jest podzielić to na mniejsze klasy. Mamy tutaj Pharmacy, PharmacyBranchAddress i PharmacyBranch. Gdy kod się rozrośnie i dojdą nowe endpointy to będzie trzeba dodać więcej danych testowych, co zaskutkuje przerośniętą klasą i nie będzie wiadomo gdzie co jest.
Można zrobić PharmacyTestData, PharmacyBranchAddressTestData itd